_includes/doc-getting-started.html

```diff
@@ -183,9 +183,10 @@ <h1 class="post-title">{% if include.truncate %}<a href="{{ page.url | absolute_
   // Do the default if there is no matching hash
   if (!foundHash) {
     var isMac = navigator.platform.toLowerCase().indexOf('mac') !== -1;
-    var isLinux = navigator.platform.toLowerCase().indexOf('linux') !== -1;
-    display('platform', isMac ? 'mac' : (isLinux ? 'linux' : 'windows'));
-    display('configuration', (isMac || isLinux) ? 'prebuilt' : 'compile');
+    var isUbuntu = navigator.platform.toLowerCase().indexOf('ubuntu') !== -1;
+    var isCentos = navigator.platform.toLowerCase().indexOf('centos') !== -1;
+    display('platform', isMac ? 'mac' : (isUbuntu ? 'ubuntu' : 'windows'));
```

Comment:
Unfortunately I don't think you get anything more than "linux" as an option here. https://stackoverflow.com/questions/19877924/what-is-the-list-of-possible-values-for-navigator-platform-as-of-today